README.md

```diff
@@ -16,4 +16,6 @@ GNomEx is a web application with a Flash client user interface and a Java backen
 - Built in ABI file viewer
 
 ### SourceForge Link 
```

Comment:
You can click "Options" in the upper right of the file display to toggle between as split or unified way of viewing the diff